stats/measure.go

```diff
@@ -15,10 +15,21 @@
 
 package stats
 
-// Measure is the interface for all measure types. A measure is required when
-// defining a view.
+// Measure represents a type of metric to be tracked and recorded.
+// For example, latency, request Mb/s, and response Mb/s are measures
+// to collect from a server.
+//
+// Each measure needs to be registered before being used.
+// Measure constructors such as NewMeasureInt64 and
+// NewMeasureFloat64 automatically registers the measure
+// by the given name.
+// Each registered measure needs to be unique by name.
+// Measures also have units that represent the unit of the
+// metric they are representing.
 type Measure interface {
 	Name() string
+	Unit() string
```

Comment:
I think this has to be a different PR. This supposed to be doc PR but also changes functionality. Please remove this change from this PR.

Reply:
Agreed, removed.